Formula/w/wgpu-native.rb

```diff
@@ -0,0 +1,39 @@
+class WgpuNative < Formula
+  desc "Native WebGPU implementation based on wgpu-core"
+  homepage "https://github.com/gfx-rs/wgpu-native"
+  url "https://github.com/gfx-rs/wgpu-native.git",
+    tag:      "v25.0.2.1",
+    revision: "af9074edf144efe4f1432b2e42c477429c4964c1"
```

Comment:
```suggestion
  url "https://github.com/gfx-rs/wgpu-native.git",
      tag:      "v25.0.2.1",
      revision: "af9074edf144efe4f1432b2e42c477429c4964c1"
```
Let's align the arguments to `url`.

Do we need a Git clone? Can we use a tarball instead?

Reply:
I tried with the tarball, but it doesn't include necessary submodules.